phpcs.xml.dist

```diff
@@ -1,12 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This was removed because there's no PHP code in this module and its it being here caused a failing linting job to be created in CI